libraries/helpers.rb

```diff
@@ -0,0 +1,35 @@
+module ElasticsearchCookbook
+  module Helpers
```

Comment:
I'm wondering if we should further separate the helpers in different modules? E.g. this one would be in `ElasticsearchCookbook::Helpers::Provider`?